### ISSUE TITLE
Support for different DTSTART/DTEND formats

### ISSUE THREAD

**Issue by StewPoll** [jasvrcek/ICS#21]
In the RFC, it specifies three main formats for datetimes.

```
   Example:  The following represents July 14, 1997, at 1:30 PM in New
      York City in each of the three time formats, using the "DTSTART"
      property.

       DTSTART:19970714T133000                   ; Local time
       DTSTART:19970714T173000Z                  ; UTC time
       DTSTART;TZID=America/New_York:19970714T133000
                                                 ; Local time and time
                                                 ; zone reference
```

Is there a method for specifying that you want to use UTC time, or Local Time with Time zone reference, when getting a calendar export instead of always using Local time/Floating time? Certain calendar programs don't like/accept local time except for All-Day events. (Outlook being the main culprit for this)

**Comment by TetraEtc** [comment]
For reference: https://tools.ietf.org/html/rfc5545#page-35

**Comment by jasvrcek** [comment]
@TetraEtc that makes sense - I'll try to add that maybe next week.

**Comment by TetraEtc** [comment]
That would be great!

For what it's worth I'd probably make them an option in the `->getStream` function. 

`->getStream($dateTimeFormat = 'UTC', $dateFormat = 'Floating')` or similar.

**Comment by bensinclair** [comment]
+1

**Comment by jasvrcek** [comment]
I'll try to get to this next week.

**Comment by bensinclair** [comment]
Great! Happy to help if you need. I'm just not sure how you'd like to do the implementation. Maybe some form os setter?

For now I've just forked and changed `src/CalendarExport.php` to use `$this->formatter->getFormattedUTCDateTime(...);` which works in Outlook, Google Calendar and other ICS readers.

**Comment by arnaudgoulpeau** [comment]
+1

I also need to specify the dtstart and dtend format, and ended up to modify CalendarExport to use getFormattedUTCDateTime instead of getFormattedDateTime, at lines 143 and 146, mainly to make it work with outlook and google calendar.

I think that the idea of @TetraEtc is great

Thank you for your all work !

**Comment by StewPoll** [comment]
Clsing as fixed in #36

**Comment by bensinclair** [comment]
Great job @StewPoll 🤘🏽
